reactapp/tailwind.config.js

```diff
@@ -1,74 +1,109 @@
+          DEFAULT: colors.blue['200'],
+          darker: colors.blue['300'],
+        },
       },
-      secondary: {
-        lighter: colors.blue['100'],
-        default: colors.blue['200'],
-        darker: colors.blue['300'],
+      textColor: {
+        orange: colors.orange,
+        primary: {
+          lighter: colors.gray['700'],
+          DEFAULT: colors.gray['800'],
+          darker: colors.gray['900'],
+        },
+        secondary: {
+          lighter: colors.gray['400'],
+          DEFAULT: colors.gray['600'],
+          darker: colors.gray['800'],
+        },
       },
-      background: {
-        lighter: colors.blue['100'],
-        default: colors.blue['200'],
-        darker: colors.blue['300'],
+      backgroundColor: {
+        primary: {
+          lighter: colors.blue['600'],
+          DEFAULT: colors.blue['700'],
+          darker: colors.blue['800'],
+        },
+        secondary: {
+          lighter: colors.blue['100'],
+          DEFAULT: colors.blue['200'],
+          darker: colors.blue['300'],
+        },
+        container: {
+          lighter: '#ffffff',
+          DEFAULT: '#fafafa',
+          darker: '#f5f5f5',
+        },
       },
-      container: {
-        lighter: colors.white,
-        default: colors.gray['100'],
-        darker: colors.gray['200'],
+      borderColor: {
+        primary: {
+          lighter: colors.blue['600'],
+          DEFAULT: colors.blue['700'],
+          darker: colors.blue['800'],
+        },
+        secondary: {
+          lighter: colors.blue['100'],
+          DEFAULT: colors.blue['200'],
+          darker: colors.blue['300'],
+        },
+        container: {
+          lighter: '#f5f5f5',
+          DEFAULT: '#e7e7e7',
+          darker: '#b6b6b6',
+        },
+      },
+      screens: {
+        sm: '640px',
+        // => @media (min-width: 640px) { ... }
+        md: '768px',
+        // => @media (min-width: 768px) { ... }
+        lg: '1024px',
+        // => @media (min-width: 1024px) { ... }
+        xl: '1280px',
+        // => @media (min-width: 1280px) { ... }
       },
-    },
-    screens: {
-      sm: '640px',
-      // => @media (min-width: 640px) { ... }
-      md: '768px',
-      // => @media (min-width: 768px) { ... }
-      lg: '1024px',
-      // => @media (min-width: 1024px) { ... }
-      xl: '1280px',
-      // => @media (min-width: 1280px) { ... }
     },
   },
   variants: {
-    borderWidth: ['responsive', 'last', 'hover', 'focus'],
-    margin: ['responsive', 'last', 'hover', 'focus'],
+    extend: {
+      borderWidth: ['last', 'hover', 'focus'],
+      margin: ['last'],
+      opacity: ['disabled'],
+      backgroundColor: ['even', 'odd'],
+    },
   },
-  plugins: [customFormsPlugin, customMinHeightPlugin],
+  plugins: [forms],
   purge: {
     content: [
-      './src/components/*.jsx',
-      './src/components/**/*.jsx',
-      '../view/frontend/templates/*.phtml',
+      '../../../../../reactapp/src/**/*.jsx',
```

Comment:
Where does these paths point to?

Reply:
This `tailwind.config.js` is not used. The original tailwindcss resides inside `src/view/` directory which is thus obviously outside the `src/reactapp`.  I put this here for IDE to recognize tailwindcss tool presence and thus its class suggestion would work again in our IDE. `tailwind.config.js` is a requirement to be present inside reactapp in order to get the class suggestion work in IDE (eg: vscode)

Reply:
I think the tailwind configuration should also be copied over to the module. Otherwise, style generation and tailwindcss class suggestions on IDE would become inconsistent. I think I should create a separate PR for that.

Reply:
Since this PR is still open, I decided to use tailwindcss setup in this PR itself. So it should work now consider the style changes added through this module also when it generates stylesheet.